flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/serializer/JsonDebeziumSchemaSerializer.java

```diff
@@ -218,6 +239,14 @@ public JsonDebeziumSchemaSerializer.Builder setNewSchemaChange(boolean newSchema
             return this;
         }
 
+        public JsonDebeziumSchemaSerializer.Builder setSchemaChangeMode(String schemaChangeMode) {
+            if (org.apache.commons.lang3.StringUtils.isEmpty(schemaChangeMode)) {
+                return this;
+            }
```

Comment:
Which schemachangemode is used when it is empty?

Reply:
JsonDebeziumSchemaChangeImplV2 is created when empty.

<img width="744" alt="image" src="https://github.com/user-attachments/assets/7ceaa96b-16d2-423f-b4a7-c88393948bf2">